### DISCUSSION TITLE
Tools for managing acode snippets

### DISCUSSION THREAD

**Discussion by rzpr** [Acode-Foundation/Acode#1022] [Show and tell]
Hi guys, i Made a tools for acode user's to manage snippets like a delete or adding. For adding snippets you can put vscode snippets raw url to [acodsnip](rezzaapr.github.io) and snippets Will be added to your snippets. I hope this can helped you all.

The preview : 

![Screenshot_2022-11-12-05-07-01-17](https://user-images.githubusercontent.com/58212770/201449349-a9fb3541-7ce4-4f2f-8feb-85a14908600c.png)

**Comment by vedxp** [comment]
Can you please provide more information about the same, including how to use and more?

**Comment by rzpr** [comment]
> Can you please provide more information about the same, including how to use and more?

Sure! You can read [Here](https://github.com/rezzaapr/rezzaapr.github.io/blob/main/acodsnip/README.md)

**Comment by vedxp** [comment]
> > Can you please provide more information about the same, including how to use and more?
> 
> Sure! You can read [Here](https://github.com/rezzaapr/rezzaapr.github.io/blob/main/acodsnip/README.md)

Thank you brother :)
